chemprop/cli/common.py

```diff
@@ -84,10 +84,10 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
         "--add-h", action="store_true", help="Whether hydrogens should be added to the mol graph."
     )
     featurization_args.add_argument(
-        "--features-generators",
+        "--molecule-featurizers",
```

Comment:
As this flag is changed, should we add a deprecation message if the old flag is used?